src/engine/enginebuffer.cpp

```diff
@@ -185,6 +185,9 @@ EngineBuffer::EngineBuffer(QString group, UserSettingsPointer pConfig,
             this, SLOT(slotEjectTrack(double)),
             Qt::DirectConnection);
 
+    m_pTrackLoaded = new ControlObject(ConfigKey(m_group, "track_loaded"), 0, 1);
```

Comment:
Why ignore-nops (0) and track (1)? (also better to use true/false instead of ints)